pages/about.js

```diff
@@ -43,13 +43,13 @@ const AboutPage = ({ lang }) => (
       <div
         className={cn(
           b('subheading', { 'mobile-only': true }),
-          b('text', { 'margin-top': 'large' })
+          styles['about-page__text--margin-top--large']
```

Comment:
why not `b`?

Reply:
can't do it here without refactoring about-page stlyles